scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -31,27 +31,32 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      disallowSleep: 'Please do not use browser.sleep() in protractor files',
-      disallowThen: 'Please do not use .then(), consider async/await instead',
       constInAllCaps: (
-        'Please make constant name “{{constName}}” are in all-caps')
+        'Please make sure that constant name “{{constName}}” are in all-caps'),
+      disallowedBrowserMethods: (
+        'Please do not use browser.{{methodName}}() in protractor files'),
+      disallowThen: 'Please do not use .then(), consider async/await instead'
     },
   },
 
   create: function(context) {
-    var checkSleepCall = function(node) {
-      var callee = node.callee;
-      if (callee.property && callee.property.name !== 'sleep') {
-        return;
-      }
+    var disallowedBrowserMethods = [
+      'sleep', 'explore', 'pause', 'waitForAngular'];
+    var disallowedBrowserMethodsRegex = (
+      `/^(${disallowedBrowserMethods.join('|')})$/`);
+    var disallowedBrowserMethodsSelector = (
+      'CallExpression[callee.object.name=browser][callee.property.name=' +
+      disallowedBrowserMethodsRegex + ']');
 
-      if (callee.object && callee.object.name === 'browser') {
-        context.report({
-          node: node,
-          loc: callee.loc,
-          messageId: 'disallowSleep'
-        });
-      }
+    var checkBrowserMethod = function(node) {
```

Comment:
Change this to `reportDisallowedBrowserMethod`!

Reply:
Done!